l10n_th_withholding_tax_multi/models/account_payment_register.py

```diff
@@ -32,7 +39,7 @@ def _onchange_payment_difference_handling(self):
                         "wt_tax_id": line.wt_tax_id.id,
                         "account_id": line.wt_tax_id.account_id.id,
                         "name": line.wt_tax_id.display_name,
-                        "amount": -line.wt_tax_id.amount / 100 * line.price_subtotal,
```

Comment:
Why change this?

Reply:
Original one is minus,
https://github.com/OCA/l10n-thailand/blob/13.0/l10n_th_withholding_tax_multi/models/account_payment.py#L35

Reply:
odoo core change

Reply:
Can you point me to where is the change? Thank you!